src/pages/starter-kit/checkout/development.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Development
+description: Learn about developing with the Adobe Commerce checkout starter kit.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# Development
+
+This guide provides basic information for software development using the checkout starter kit.
+
+## Prerequisites
+
+You must install or have access to the following prerequisites to develop with the Adobe Commerce checkout starter kit:
+
+- [Node.js](https://nodejs.org/) version 22. If you have Node Version Manager (`nvm`) installed, you can run the following command to install and use the required version:
+
+  ```bash
+  cat .nvmrc | nvm install && nvm use
```

Comment:
Using `cat .nvmrc` assumes you already cloned the repo, which is incorrect.
Could you change to just
```bash
nvm install 22 && nvm use
```